ansible/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: "Build hosts file"
+  lineinfile: dest=/etc/hosts regexp='.*{{ item }}$' line="{{item}}" state=present
```

Comment:
Unclear from [the docs] whether `line` will be inserted if `regexp` isn't found. 

  [1]: http://docs.ansible.com/ansible/lineinfile_module.html